spiderfoot/db.py

```diff
@@ -144,6 +144,7 @@ class SpiderFootDb:
         ['DOMAIN_WHOIS', 'Domain Whois', 1, 'DATA'],
         ['EMAILADDR', 'Email Address', 0, 'ENTITY'],
         ['EMAILADDR_COMPROMISED', 'Hacked Email Address', 0, 'DESCRIPTOR'],
+        ['EMAILADDR_DISPOSABLE', 'Disposable Email Address', 0, 'ENTITY'],
```

Comment:
Should be `DESCRIPTOR` since the disposable state of the email is describing the `EMAILADDR` entity.